src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/cessations/BeneficialOwnerCessation.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.cessations;
+
+import java.time.LocalDate;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public abstract class BeneficialOwnerCessation extends Cessation {
+
+    private static final String CHANGE_NAME = "ceasePSCAppointment";
```

Comment:
Should this not be 'pscCessation' by default according to the mapping document?

Reply:
Agreed

Reply:
I had changed this and the merge nuked it. Sigh